package.json

```
@@ -11,13 +11,15 @@
     "build": "npm run clean && tsc -d && cp src/*.mjs .",
```

Er zou ook een action moeten zijn die checkt of `yarn build` faalt
```suggestion
    "build": "npm run clean && tsc -d && cp src/mjs/*.mjs .",
```